src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -609,6 +625,22 @@ public override ushort UInt16Value
                 }
             }
 
+            public override int Int32Value
```

Comment:
>public override int Int32Value [](start = 12, length = 30)

Should we also override UInt32Value property? In general, I find semantics of properties that return unsigned values unclear. For example, the following code prints two different numbers
```
                short x = -1;
                System.Console.WriteLine((uint)x);
                System.Console.WriteLine((uint)(ushort)x);
```
Which one do we want to be the result of UInt32Value property?